vector/src/main/java/im/vector/activity/SimpleFragmentActivity.kt

```diff
@@ -51,18 +50,29 @@ open class SimpleFragmentActivity : MXCActionBarActivity() {
      * Displays a progress indicator with a message to the user.
      * Blocks user interactions.
      */
-    fun showWaitingView(status: String) {
-        waitingStatusText.text = status
-        waitingStatusProgress.isVisible = false
-        showWaitingView()
-    }
+    fun updateWaitingView(data: WaitingViewData?) {
```

Comment:
no more animation on status update?

Reply:
No, but the dialog width is now set to 75% of the screen in portrait and 50% in Landscape, to limit dialog size change